server/storage/endpoint/config.go

```diff
@@ -74,3 +76,25 @@ func (se *StorageEndpoint) SaveScheduleConfig(scheduleName string, data []byte)
 func (se *StorageEndpoint) RemoveScheduleConfig(scheduleName string) error {
 	return se.Remove(scheduleConfigPath(scheduleName))
 }
+
+// LoadStoreConfig loads the config of store.
+func (se *StorageEndpoint) LoadStoreConfig(cfg interface{}) (bool, error) {
```

Comment:
Can it not persist? TiKV as a source of truth.

Reply:
In past, the trigger of the updated config is `PutStore`, and this api will not be called if pd restart or transfer leader.